example/Makefile

```
@@ -0,0 +1,5 @@
+build: chain-indexing-app
+chain-indexing-app:
+	go build ./app/example-app/
+clean:
+	rm example-app
```

```suggestion
	rm -i example-app
```
how about add a prompt before removing?

---

Though not sure if it is violating the Makefile common practice

---

Shall be okay